src/main/locales/da/locales.h

```diff
@@ -0,0 +1,185 @@
+/*
```

Comment:
I would also consider naming these files: `locale.h` not `locales.h`

Reply:
right, done